src/test/groovy/com/google/plugins/ProtobufAndroidPluginTest.groovy

```diff
@@ -15,10 +15,30 @@ class ProtobufAndroidPluginTest extends Specification {
     def mainProjectDir = ProtobufPluginTestHelper.prepareTestTempDir('testProjectAndroid')
     ProtobufPluginTestHelper.copyTestProjects(mainProjectDir, 'testProject', 'testProjectLite', 'testProjectAndroid')
 
+    // Add android plugin to the test root project so that Gradle an resolve
+    // classpath correctly.
+    new File(mainProjectDir, "build.gradle") << """
+buildscript {
+    def androidPluginVersion = System.properties.get("ANDROID_PLUGIN_VERSION") ?: "2.2.0"
+    repositories {
+        maven { url "https://plugins.gradle.org/m2/" }
+        if (androidPluginVersion.startsWith("3.")) {
+            google()
+        }
+    }
+    dependencies {
+        classpath "com.android.tools.build:gradle:$androidPluginVersion"
+    }
+}
```

Comment:
This snippet is already in `testProjectAndroid/build.gradle`. Should it be removed?